includes/Runtime.php

```diff
@@ -73,7 +73,7 @@ public function register_runtime() {
 		global $wp_version;
 		\wp_register_script(
 			'nfd-runtime',
-			null,
+			'',
```

Comment:
Looks like yes, either a string or false is required for the parameter. I think we should use false here though, since it is more clear. While technically I think both would work, but may as well be clear for ourselves that there's no path with a false. We've seen stray scripts try to load from our plugin dir as a file with no path, I don't think this is the cause, but not entirely sure where those are coming from.
```suggestion
			false,
```